server-spi-private/src/main/java/org/keycloak/events/EventType.java

```diff
@@ -132,7 +132,10 @@
     TOKEN_EXCHANGE_ERROR(true),
 
     PERMISSION_TOKEN(true),
-    PERMISSION_TOKEN_ERROR(false);
+    PERMISSION_TOKEN_ERROR(false),
+
+    DELETE_ACCOUNT(true),
+    DELETE_ACCOUNT_ERROR(true);
```

Comment:
I know that none of the rest of the EventTypes have JavaDoc, but it'd be good to add it for new additions. See [KEYCLOAK-15172](https://issues.redhat.com/browse/KEYCLOAK-15172) for more details.